src/components/Textarea/Textarea.tsx

```diff
@@ -47,9 +49,25 @@ export const Textarea = ({
     resizeable = true,
     onFocus,
     selectable = false,
+    focusOnMount,
+    onEnterPressed,
 }: TextareaProps): ReactElement => {
     const Component = autosize ? TextareaAutosize : 'textarea';
 
+    const textareaElement = useRef<HTMLTextAreaElement | null>(null);
+
+    useEffect(() => {
+        setTimeout(() => {
```

Comment:
Why do we need the setTimeout? It seems unnecessary to me, especially with a delay set to 0.